java/compaction-core/src/main/java/sleeper/compaction/job/status/CompactionJobStatus.java

```diff
@@ -98,24 +100,17 @@ private Instant firstTime() {
     }
 
     private Instant lastTime() {
-        if (jobRunList.isEmpty()) {
-            return createdStatus.getUpdateTime();
-        }
-        return getLatestJobRun().getLatestUpdateTime();
-    }
-
-    private CompactionJobRun getLatestJobRun() {
-        return jobRunList.get(0);
+        return jobRuns.lastTime().orElse(createdStatus.getUpdateTime());
     }
 
-    public List<CompactionJobRun> getJobRuns() {
-        return jobRunList;
+    public List<ProcessRun> getJobRunList() {
+        return jobRuns.getRunList();
```

Comment:
Shall we keep the plural name, ie. getJobRuns, getRuns? I think it's fine to delegate to ProcessRuns without letting you get it directly.